utils/ansible_conf/custom_attributes_basic_script.yml

```diff
@@ -0,0 +1,17 @@
+- hosts: localhost
```

Comment:
**REQUIRED:** I think it makes sense to move this file to the data folder, there is a data file loader you can see in the code base, if you need help finding it gimme a shout, but this doesn't belong in utils.